modules/Bio/EnsEMBL/Compara/PipeConfig/UpdateReferenceDatabase_conf.pm

```diff
@@ -86,20 +87,42 @@ sub default_options {
         'create_all_mlss_exe' => $self->check_exe_in_ensembl('ensembl-compara/scripts/pipeline/create_all_mlss.pl'),
         'allowed_species_file'  => $self->check_file_in_ensembl('ensembl-compara/conf/' . $self->o('division') . '/allowed_species.json'),
         'xml_file'            => $self->check_file_in_ensembl('ensembl-compara/conf/' . $self->o('division') . '/mlss_conf.xml'),
+
+        # whole dc options
+        'datacheck_groups' => ['compara_references'],
+        'db_type'          => ['compara'],
+        'output_dir_path'  => $self->o('pipeline_dir') . '/datachecks/',
+        'overwrite_files'  => 1,
+        'failures_fatal'   => 1, # no DC failure tolerance
+        'ref_dbname'       => 'ensembl_compara_references', # to be manually passed in init if differs
```

Comment:
I think keep it here, because if we decide later on to do the re-wiring mentioned, it will be easier to keep track and change if they are all together?